fractal_tasks_core/create_ome_zarr.py

```diff
@@ -216,9 +212,24 @@ def create_ome_zarr(
                 pixel_size_x = site_metadata["pixel_size_x"][0]
                 bit_depth = site_metadata["bit_depth"][0]
         except FileNotFoundError:
+            # FIXME: Why are we letting this pass? If no metadata file is 
```

Comment:
This was probably for testing with datasets with no available mlf/mrf files.
Let's make it fail with an error, and in case it's needed we can fix the tests.

Reply:
Ok, I removed the try/catch for it. Let's see if it makes new tests fail

Reply:
They are still passing.